tests/foreman/cli/test_contentviews.py

```diff
@@ -1088,14 +1019,18 @@ def test_cv_promote_composite(self):
             u'id': con_view['id'],
             u'content-view-version-id': version1_id,
         })
-        self.assertEqual(result.return_code, 0,
-                         "Repo was not associated to selected CV")
+        self.assertEqual(
+            result.return_code, 0,
+            "Repo was not associated to selected CV"
```

Comment:
You have removed the third parameter, the error message that gets displayed to the user, in other parts of the code, so perhaps you should remove this one here too?